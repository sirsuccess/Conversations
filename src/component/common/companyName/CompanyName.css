.CompanyName {
    display: flex;
    align-items: center;
    /* justify-content: space-between; */
    /* width: 250px; */
    color: #fff;
    font-size: 14px;
    font-family: IBMPlexSans;
    font-weight: 500;
}

.circle {
    border-radius: 50%;
    background-color: rgba(255, 255, 255, 0.3);
    height: 22px;
    width: 25px;
    margin-right: 1em;
    padding-top: 0.2em;
    /* text-align: center; */
}

.fa-chevron-down {
    color: #000;
    margin-left: 1em;
    cursor: pointer;
    font-size: 12px;
    margin-top: 0.3em;
}