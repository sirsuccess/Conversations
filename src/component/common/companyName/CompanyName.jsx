import React from "react";

import "./CompanyName.css";

function CompanyName() {
  return (
    <div className="CompanyName">
      <div className="circle">S</div>
      Organisation Name <i class="fa fa-chevron-down"></i>
    </div>
  );
}

export default CompanyName;
