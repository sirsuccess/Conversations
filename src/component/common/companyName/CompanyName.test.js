import React from "react";
import renderer from "react-test-renderer";

import CompanyName from "./CompanyName";

it("renders correctly when called ", () => {
  const tree = renderer.create(<CompanyName />).toJSON();
  expect(tree).toMatchSnapshot();
});
