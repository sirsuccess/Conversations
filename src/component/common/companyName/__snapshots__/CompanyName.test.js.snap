// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<div
  className="CompanyName"
>
  <div
    className="circle"
  >
    S
  </div>
  Organisation Name 
  <i
    className="fa fa-chevron-down"
  />
</div>
`;
